src/main/kotlin/uk/gov/justice/digital/hmpps/courtregister/resource/CourtResource.kt

```diff
@@ -241,6 +241,36 @@ class CourtResource(
     @Schema(description = "Contact ID", example = "11111", required = true) @PathVariable contactId: Long
   ): ContactDto =
     contactService.findById(courtId, buildingId, contactId)
+
+  @GetMapping("/id/{courtId}/subcodes", "/id/{courtId}/subcodes/{subCode}")
```

Comment:
shouldn't be `buildings/courtId/{courtId}/main`   and `buildings/courtId/{courtId}/subcode/{subcode}`  

(and this under `court` so I guess `/court/building/.....'